corehq/apps/userreports/reports/specs.py

```diff
@@ -369,6 +370,13 @@ class _CaseExpressionColumn(ReportColumn):
     _agg_column_type = None
 
     def get_column_config(self, data_source_config, lang):
+        if not data_source_config.is_static and not settings.UNIT_TESTING:
```

Comment:
could potentially change this to `DEBUG` so that you don't need to be in nosetests for it to work locally